smitch88/bmorearoundtown_v2
# Events CRUD front end

# What

Events Table, Create/Update Event View, Event Details view -- all routes are private, flow is correct. There are some missing links based on some information I am lacking but mostly there. 

# Changes

Just added all the views. Also added sagas and for CRUD, did not add reducers because unsure of the final events data structure. All pages are fully responsive. 

# Direction

Need to add some little styling things as they come. Responsive menu for event details view -- Would be nice to add some icons here and there. Waiting for feedback on this iteration. Note, this is not yet configured to make API calls, -- still using mock data. 

# How to test

got to /cp then login --> click events link, select a specific event to see details view or select create new event to see create view --> from event details you can go to update event view.
